packages/valko-ui-components/src/types/Textarea.ts

```diff
@@ -1,6 +1,6 @@
-import type { DefaultComponent } from './common'
+import type { DefaultComponent, AriaAttributes } from './common'
 
-export interface TextareaProps extends DefaultComponent {
+export interface TextareaProps extends DefaultComponent, Pick<AriaAttributes, 'aria-label' | 'aria-labelledby' | 'aria-describedby' | 'aria-invalid' | 'aria-required'>{
```

Comment:
Check if this is shorter using Omit instead of Pick, since you are picking a lot of props